lm75bd/lm75bd.c

```diff
@@ -26,8 +26,16 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
+  error_code_t errCode = ERR_CODE_INVALID_ARG;
+  if (temp == NULL) return errCode; //If arguments are problematic return early
+  uint8_t buf[2] = {0x00U, 0x00U}; 
+  uint8_t readBuf = 0x00;
+  RETURN_IF_ERROR_CODE(i2cSendTo(devAddr, &readBuf,1));
```

Comment:
Sending the contents of `readBuf` is confusing. I'd recommend changing that variable name. It'd make more sense for the buffer passed into `i2cReceiveFrom` to be called `readBuf`

Also, we have a #define for each register address. For example, you can find the configuration register address (LM75BD_REG_CONF) at the top of this file.